client/src/main/java/io/pravega/client/stream/TransactionInfo.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Pravega Authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.client.stream;
+
+import java.util.UUID;
+
+/**
+ * TransactionInfo holds the information of transaction.
+ *  It contains streamName, transactionId and transactionStatus
```

Comment:
Unnecessary extra space at the start of the comment.

Reply:
Deleted extra space.